ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -563,12 +578,8 @@ if [ $verify = precip ] ; then
   done # end of metplus_jobs loop
 
   # Indicate all tasks are completed
-  >$WORK/stats_completed
-  echo "stats are completed" >> $WORK/stats_completed
-
-  if [ $SENDCOM = YES ] ; then
-    cp $WORK/stats_completed $COMOUTsmall_precip
-  fi
+  >$COMOUTsmall_precip/stats_completed
```

Comment:
It looks like this is writing directly to COMOUTsmall. We can only copy files there.

Can the restart file get written to DATA, and then `if [ $SENDCOM = YES ] ; then` copy to COMOUTsmall?